db/database.go

```diff
@@ -644,7 +644,7 @@ func NewDatabaseContext(ctx context.Context, dbName string, bucket base.Bucket,
 		// No cleanup necessary, stop heartbeater above will take care of it
 	}
 
-	dbContext.ResyncManager = NewResyncManager(bucket)
+	dbContext.ResyncManager = NewResyncManagerDCP(bucket)
```

Comment:
@adamcfraser we potentially need to discuss if we are going to toggle between `NewResyncManager` and `NewResyncManagerDCP`. I remember you wanted to do a performance test. We should discuss that

Reply:
Yes, let's add an unsupported database level config option to run with the old, non-DCP version.